src/common/decorators/swagger.ts

```diff
@@ -115,22 +115,39 @@ export function ApiTransactionResponse(
   );
 }
 
+type FailedTransactionResponseCodes =
+  | HttpStatus.NOT_FOUND
+  | HttpStatus.BAD_REQUEST
+  | HttpStatus.UNPROCESSABLE_ENTITY;
+
 /**
- * A helper that combines responses for SDK Error like `BadRequestException`, `NotFoundException`, `UnprocessableEntityException`, `InternalServerErrorException`
+ * A helper that combines responses for SDK Errors like `BadRequestException`, `NotFoundException`, `UnprocessableEntityException`
```

Comment:
```suggestion
 * A helper that combines responses for HTTP Errors like `BadRequestException`, `NotFoundException`, `UnprocessableEntityException`
```

src/transactions/transactions.util.ts has `handleSdkError` that transforms them so Nest doesn't return a 500